### ISSUE TITLE
Uncaught Error: Too few arguments to function nwdcx_throwable(), 1

### ISSUE THREAD

**Issue by khromov** [nawawi/docket-cache#10]
When used with PHP8, I get the following when trying to change a setting on the Configuration page with plugin version `21.02.02`

```
Fatal error: Uncaught Error: Too few arguments to function nwdcx_throwable(), 1 passed in /var/www/html/wp-content/plugins/docket-cache/includes/src/WpConfig.php on line 220 and exactly 2 expected
in /var/www/html/wp-content/plugins/docket-cache/includes/compat.php on line 208

Anropsstack:

    nwdcx_throwable()
    wp-content/plugins/docket-cache/includes/src/WpConfig.php:220
    Nawawi\D\WpConfig::has()
    wp-content/plugins/docket-cache/includes/src/ReqAction.php:273
    Nawawi\D\ReqAction::run_action()
    wp-content/plugins/docket-cache/includes/src/ReqAction.php:313
    Nawawi\D\ReqAction::parse_action()
    wp-content/plugins/docket-cache/includes/src/ReqAction.php:29
    Nawawi\D\ReqAction::Nawawi\D\{closure}()
    wp-includes/class-wp-hook.php:287
    WP_Hook::apply_filters()
    wp-includes/class-wp-hook.php:311
    WP_Hook::do_action()
    wp-includes/plugin.php:484
    do_action()
    wp-admin/admin.php:237
```

**Comment by nawawi** [comment]
Hi,

Thanks for the report. Seems like a missing argument for nwdcx_throwable at WpConfig::has method.

![image](https://user-images.githubusercontent.com/1203635/115254820-4d417580-a160-11eb-917b-8a657113ad34.png)

Will update the release as soon as possible.

Thanks.

**Comment by nawawi** [comment]
this issue has been fixed in version 21.02.03. thanks.
